torchrl/modules/models/models.py

```diff
@@ -488,8 +523,10 @@ def __init__(
         out_features_value: int = 1,
         mlp_kwargs_feature: Optional[dict] = None,
         mlp_kwargs_output: Optional[dict] = None,
+        device: DEVICE_TYPING = "cpu",
     ):
-        super(DuelingMlpDQNet, self).__init__()
+        super().__init__()
+        self.device = torch.device(device)
```

Comment:
the problem with this is that when calling `net.to(device)` this attribute won't change